### PR TITLE
Miarka deployment

### PR DESCRIPTION
**Pull request by b97pla**
- Replaces the obvious references to Irma-specific paths with the corresponding Miarka paths

### CODE REVIEW

File: run_hs_metrics.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ METRICS=$5
 OUTDIR="${ANALYSISDIR}/results/Reports/HsMetrics"
 
 # singularity container used by sarek, provides CollectHsMetrics
-CONTAINER=/lupus/ngi/containers/sarek/nfcore-sarek-2.6.1.img
+CONTAINER=/vulpes/ngi/containers/sarek/nfcore-sarek-2.6.1.img
```
Comment (mariya) [inline run_hs_metrics.sh]: This path doesn't seem to exist? I did find `/vulpes/ngi/containers/sarek/nfcore-sarek-2.7.img` though.

File: statdump_to_json.pl [hunk 2]
```diff
@@ -3,7 +3,7 @@
 use strict;
 use JSON;
 
-use lib "/lupus/ngi/production/latest/sw/arteria/siswrap_venv/deps/sisyphus/sisyphus-latest/lib";
+use lib "/vulpes/ngi/production/latest/sw/arteria/siswrap_venv/deps/sisyphus/sisyphus-latest/lib";
```
Comment (mariya) [inline statdump_to_json.pl]: This path doesn't exist. It seems to be a sisyphus thing... can we just remove this script?

File: twist_exome_38_template.sbatch [hunk 3]
```diff
@@ -5,16 +5,16 @@
 #SBATCH -N 1
 #SBATCH -n 1
 #SBATCH -t 10-00:00:00
-#SBATCH -o "/lupus/proj_nobackup/ngi2016001/NGI/ANALYSIS/=PROJECT=/TwistExome.%j.out"
-#SBATCH -e "/lupus/proj_nobackup/ngi2016001/NGI/ANALYSIS/=PROJECT=/TwistExome.%j.err"
-#SBATCH -D "/lupus/proj_nobackup/ngi2016001/NGI/ANALYSIS/=PROJECT=/"
+#SBATCH -o "/proj/ngi2016001/nobackup/NGI/ANALYSIS/=PROJECT=/TwistExome.%j.out"
```
Comment (mariya) [inline twist_exome_38_template.sbatch:8]: There are a whole lot of paths in this file that do not exist. My opinion is that we should just remove these scripts; they still exist in the history if someone should need them. If you don't want to remove them, I suggest we at least leave them as is to make it clear that they have not been updated/tested to work on miarka.

**Review by mariya — CHANGES_REQUESTED** [review]

File: run_hs_metrics.sh (hunk 1 above)
Comment (b97pla) [reply run_hs_metrics.sh]: true

File: statdump_to_json.pl (hunk 2 above)
Comment (b97pla) [reply statdump_to_json.pl]: yes, that should be ok

File: twist_exome_38_template.sbatch (hunk 3 above)
Comment (b97pla) [reply twist_exome_38_template.sbatch:8]: I think this script is needed, I'll review the paths again and see if I can fix

Comment (b97pla) [reply twist_exome_38_template.sbatch:8]: I only found one path that I changed, the other are paths that are created in the process when the script is used or reference data that should be moved to the place pointed to in the script.

**Review by mariya — APPROVED** [review]

**Pull request merged**